nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -603,6 +603,10 @@ sub multiLineList {
   # Enable CUPS to print documents.
   # services.printing.enable = true;
 
+  # Enable sound.
+  # sound.enable = true;
+  # hardware.pulseaudio.enable = true;
```

Comment:
BTW, shouldn't `hardware.pulseaudio.enable = true` imply `sound.enable = true`? Or perhaps the other way around, if we want Pulseaudio to be the default sound system.